lib/screens/models/consultation_model.dart

```diff
@@ -16,12 +16,12 @@ class ConsultationModel {
   final User user;
   PatientModel? patient;
   OmrsLocation? location;
-
   ConsultationStatus status = ConsultationStatus.none;
   DateTime? startTime;
   DateTime? lastUpdateAt;
   List<ConditionModel> problemList = [];
   List<ConditionModel> diagnosisList = [];
+  List<String> investigationList = [];
```

Comment:
investigation can not be modeled as String. It requires an order model, which will have whats orderable, who ordered it, date of order,  whether to be done on a specific date, instruction for the investigation etc.